cmd/panfs.go

```diff
@@ -776,11 +780,8 @@ func (fs *PANFSObjects) DeleteBucket(ctx context.Context, bucket string, opts De
 
 	globalBucketMetadataCache.Delete(bucket)
 	if fs.configAgent != nil {
-		noLockID := ""
-		if err = fs.configAgent.DeleteObject(
-			pathJoin(panfsBucketListPrefix, bucket), noLockID); err != nil {
-			return toObjectErr(err, bucket)
-		}
+		// NOTE:llorens: do nothing, deleteBucketMetadata() will do
```

Comment:
Can we  get rid of that if-else block and just check `if configAgent == nil {...}`?

Reply:
I even think that we can remove that `if-else` completely (it seems that there is no need to delete bucket metadata directory manually as `deleteBucketMetadata` does it). At least it is working well for local bucket metadata store but I think it should be fine for configagent as well